DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/ManageCourseController.cs

```diff
@@ -136,5 +199,83 @@ EditCourseOptionsViewModel editCourseOptionsViewModel
             courseService.UpdateCourseOptions(courseOptions, customisationId);
             return RedirectToAction("Index", "ManageCourse", new { customisationId });
         }
+
+        private void ValidateCustomisationName(string customisationName, EditCourseDetailsViewModel model)
+        {
+            if (customisationName.Length > 250)
+            {
+                ModelState.AddModelError(
+                    nameof(EditCourseDetailsViewModel.CustomisationNameSuffix),
+                    "Course name must be 250 characters or fewer, including any additions"
+                );
+            }
+            else if (courseService.DoesCourseNameExistAtCentre(
+                model.CustomisationId,
+                customisationName,
+                model.CentreId,

```

Comment:
I'm also not convinced that CentreId should be posted back to the endpoint to be used like this, we should have the "Can user access this course" attribute, and then just get the centre Id off the User

Reply:
removed centreId from the FormData so it is not posted

Reply:
Still posting it to nothing in the view (it has a hidden)

Reply:
Removed the hidden